test/unit/test_generator.py

```diff
@@ -3,18 +3,19 @@
 
 import pytest
 from pdf_generator.generator import (
-    LOGO_FILE, MAIN_TEMPLATE, META_TEMPLATE, PLUGIN_TEMPLATE_BLUEPRINT, copy_fact_image, create_report_filename,
-    create_templates, execute_latex, render_analysis_templates
+    LOGO_FILE, MAIN_TEMPLATE, META_TEMPLATE, copy_fact_image, create_report_filename,
+    create_templates, execute_latex
 )
+from test.data.test_dict import TEST_DICT, META_DICT
 
 
 class MockEngine:
     def __init__(self, *_, **__):
         pass
 
     @staticmethod
-    def render_main_template(analysis, meta_data):
-        return '{}\n{}'.format(json.dumps(analysis), json.dumps(meta_data))
+    def render_main_template(analysis):
+        return '{}'.format(json.dumps(analysis))
```

Comment:
`'{}'.format(..)` is redundant here (the output of `json.dumps()` is already a string)
```suggestion
        return json.dumps(analysis)
```